node/popn.go

```diff
@@ -225,9 +224,9 @@ func New(ctx context.Context, opts Options) (*node, error) {
 	if err != nil {
 		return nil, err
 	}
-	if opts.PrivKey != "" {
-		nd.importAddress(opts.PrivKey)
-	}
+
+	defaultAddress := nd.exch.Wallet().DefaultAddress()
+	fmt.Println("==> Default FIL address loaded: ", defaultAddress)
```

Comment:
I would start with the verb to stay consistent with other feedback messages
```suggestion
	fmt.Println("==> Loaded default FIL address: ", defaultAddress)
```